src/components/typography/text/text.mod.css

```diff
@@ -12,97 +12,110 @@
   --font-color: var(--color-black);
 }
 
-h1 {
+h1.headline {
   font-size: var(--font-h1);
 }
 
-h2 {
+h2.headline {
   font-size: var(--font-h2);
 }
 
-h3 {
+h3.headline {
   font-size: var(--font-h3);
 }
 
-h4 {
+h4.subheadline {
   font-size: var(--font-h4);
   font-weight: normal;
 }
 
-h5 {
+h5.subheadline {
```

Comment:
Could we remove those `h1` prefixes from the selectors? Our JS code should ensure that they're applied to the right elements. I wouldn't consider it part of the CSS's job.

Reply:
We still need it for the font size though right? either that or we need two different classnames.

Reply:
Ah yeah, I missed that part. I'm okay with either approach then 👍